packages/contracts/development/contracts/RealityETHFreezableExample_ERC20-4.0.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.20;
+
+import {RealityETHFreezable_ERC20} from "./RealityETHFreezable_ERC20.sol";
+
+/*
+ * This contract is designed for testing the abstract contract RealityETHFreezable_ERC20.
+ * It should not be used as is because anyone can call setFreezeTimestamp.
+ */
+
+// solhint-disable-next-line contract-name-camelcase
+contract RealityETHFreezableExample_ERC20_v4_0 is RealityETHFreezable_ERC20 {
```

Comment:
I would put these test files into an own folder. I think its then much easier to navigate

Reply:
Will consider rearranging later, might move the interfaces as well. Won't do it hear as it has implications for other test/deployment tooling.